dts/msm8916-samsung.dts

```diff
@@ -24,6 +24,17 @@
 		lk2nd,match-bootloader = "A500FU*";
 	};
 
+	j5xlte-eur {
+		model = "Samsung Galaxy J5 2016 LTE (EUR)";
```

Comment:
Maybe
```suggestion
		model = "Samsung Galaxy J5 2016 (SM-J510FN)";
```
is more clear than LTE+EUR? I could be convinced otherwise, you can choose.